### PR TITLE
Update Elasticsearch installation docs

### PR DESCRIPTION
**Pull request by ezodude**
- Instructions follow the new values.yaml.
- Removed optional index type.

## Description

Changes proposed in this pull request:

- Instructions follow the new values.yaml.
- Removed optional index type.

## Related issue(s)

See also [Release 0.13 and promote it](https://github.com/kubeshop/botkube/issues/595)

### CODE REVIEW

File: content/installation/ElasticSearch/_index.md [hunk 1]
```diff
@@ -20,17 +20,24 @@ toc = true
...
+  
+  $ helm install --version v0.13.0 botkube --namespace botkube --create-namespace \
+  --set communications.default-group.elasticsearch.enabled=true \
+  --set communications.default-group.elasticsearch.server=<ELASTICSEARCH_ADDRESS> \
+  --set communications.default-group.elasticsearch.username=<ELASTICSEARCH_USERNAME> \
+  --set communications.default-group.elasticsearch.password=<ELASTICSEARCH_PASSWORD> \
+  --set communications.default-group.elasticsearch.indices.default.name=<ELASTICSEARCH_INDEX_NAME> \
+  --set communications.default-group.elasticsearch.indices.default.shards=<ELASTICSEARCH_INDEX_SHARDS> \
+  --set communications.default-group.elasticsearch.indices.default.replicas=<ELASTICSEARCH_INDEX_REPLICAS> \
+  --set settings.clusterName=<CLUSTER_NAME> \
```
Comment (mszostok) [inline content/installation/ElasticSearch/_index.md:39]:
```suggestion
  --set communications.default-group.elasticsearch.enabled=true \
  --set communications.default-group.elasticsearch.server=${ELASTICSEARCH_ADDRESS} \
  --set communications.default-group.elasticsearch.username=${ELASTICSEARCH_USERNAME} \
  --set communications.default-group.elasticsearch.password=${ELASTICSEARCH_PASSWORD} \
  --set communications.default-group.elasticsearch.indices.default.name=${ELASTICSEARCH_INDEX_NAME} \
  --set communications.default-group.elasticsearch.indices.default.shards=${ELASTICSEARCH_INDEX_SHARDS} \
  --set communications.default-group.elasticsearch.indices.default.replicas=${ELASTICSEARCH_INDEX_REPLICAS} \
  --set settings.clusterName=${CLUSTER_NAME} \
```

File: content/installation/ElasticSearch/_index.md [hunk 2]
```diff
@@ -20,17 +20,24 @@ toc = true
...
+  --set image.tag=v0.13.0 \
```
Comment (mszostok) [inline content/installation/ElasticSearch/_index.md:40]: this can be removed, as this will be a default 👍

**Review by mszostok — APPROVED** [review]
just a minor comment, besides that LGMT 👍

File: content/installation/ElasticSearch/_index.md (hunk 1 above)
Comment (ezodude) [reply content/installation/ElasticSearch/_index.md:39]: Of course.

File: content/installation/ElasticSearch/_index.md (hunk 2 above)
Comment (ezodude) [reply content/installation/ElasticSearch/_index.md:40]: of course.

**Comment by mszostok** [comment]
Please hold off with merging before 0.13 is out :)

File: content/installation/ElasticSearch/_index.md [hunk 3]
```diff
@@ -20,17 +20,17 @@ toc = true
...
+  export ELASTICSEARCH_USERNAME={elasticsearch_username}
+  export ELASTICSEARCH_PASSWORD={elasticsearch_password}
+  export ELASTICSEARCH_INDEX_NAME={elasticsearch_index_name}  
+  
+  $ helm install --version v0.13.0 botkube --namespace botkube --create-namespace \
+  --set communications.default-group.elasticsearch.enabled=true \
+  --set communications.default-group.elasticsearch.server=${ELASTICSEARCH_ADDRESS} \
+  --set communications.default-group.elasticsearch.username=${ELASTICSEARCH_USERNAME} \
+  --set communications.default-group.elasticsearch.password=${ELASTICSEARCH_PASSWORD} \
+  --set communications.default-group.elasticsearch.indices.default.name=${ELASTICSEARCH_INDEX_NAME} \
```
Comment (mszostok) [inline content/installation/ElasticSearch/_index.md:33]:
```suggestion
  export CLUSTER_NAME={cluster_name}
  export ELASTICSEARCH_ADDRESS={elasticsearch_address}
  export ELASTICSEARCH_USERNAME={elasticsearch_username}
  export ELASTICSEARCH_PASSWORD={elasticsearch_password}
  export ELASTICSEARCH_INDEX_NAME={elasticsearch_index_name}  
  
  $ helm install --version v0.13.0 botkube --namespace botkube --create-namespace \
  --set communications.default-group.elasticsearch.enabled=true \
  --set communications.default-group.elasticsearch.server=${ELASTICSEARCH_ADDRESS} \
  --set communications.default-group.elasticsearch.username=${ELASTICSEARCH_USERNAME} \
  --set communications.default-group.elasticsearch.password=${ELASTICSEARCH_PASSWORD} \
  --set communications.default-group.elasticsearch.indices.default.name=${ELASTICSEARCH_INDEX_NAME} \
  --set settings.clusterName=${CLUSTER_NAME} \
```

**Pull request merged**